docker-bake.hcl

```diff
@@ -0,0 +1,64 @@
+group "default" {
+  targets = ["image-local"]
+}
+
+// Special target: https://github.com/docker/metadata-action#bake-definition
+target "docker-metadata-action" {
+  tags = ["faq:local"]
+}
+
+group "validate" {
+  targets = ["lint"]
+}
+
+target "lint" {
+  target = "lint"
+  output = ["type=cacheonly"]
+}
+
+target "test" {
+  target = "test-coverage"
+  output = ["."]
+}
+
+target "artifact" {
+  target = "artifacts"
+  secret = ["id=GITHUB_TOKEN,env=GITHUB_TOKEN"]
```

Comment:
We can manage secrets at build time. Here the GITHUB_TOKEN we be used in the `artifact` stage in the Dockerfile.